test/behaviors/t_for_all.clj

```diff
@@ -33,26 +33,24 @@
 
 (def pass-count (state/output-counters:midje-passes))
 (for-all
-  [strictly-pos gen/s-pos-int
-   any-integer  gen/int]
+  [strictly-pos gen/s-pos-int]
   {:seed 3510160943861}
```

Comment:
The use a hard coded seed to force a logically nonsensical test to pass was just me being sloppy. I like the cleanup you've done.

Can you also remove the `{:seed ..}` options argument from these tests as they are no longer needed?